### ISSUE TITLE
release new version

### ISSUE THREAD

**Issue by hauntingEcho** [cloudtools/stacker#759]
The most current version of Stacker was released 16 months ago, and there have been important bugfixes since - in particular, #731 is important to my team.  Will there be a new release soon?

**Comment by troyready** [comment]
@hauntingEcho an alternative you can try while waiting is [runway](https://github.com/onicagroup/runway), another python project that forked its CloudFormation support off Stacker (with additional features and other fixes like the one you mentioned included).

Its scope is a bit broader (essentially, orchestrating multiple Stacker deployments alongside other cloud deployment tools like Terraform and Serverless Framework), but it can just be used as an opinionated Stacker replacement as well.

E.g., assuming you had a directory like:

```
mystackerconfigfile.yml
mystacker-dev-env.env
```
and wanted to deploy it to us-east-2, 
```
mv mystacker-dev-env.env dev-us-east-2.env
cat << EOF > runway.yml
---
deployments:
  - modules:
      - ./
    regions:
      - us-east-2
EOF
runway deploy --deploy-environment dev
```

**Comment by hauntingEcho** [comment]
closed via #760
